build-system/compile/bundles.config.js

```diff
@@ -753,7 +753,7 @@ exports.extensionBundles = [
   },
   {
     name: 'amp-next-page',
-    version: ['0.1', '0.2'],
+    version: ['0.1', '1.0'],
     latestVersion: '0.1',
```

Comment:
Sanity check: Does this line need to change?

Reply:
No, `1.0` is still experimental 😄  !